client/app/scripts/actions/app-actions.js

```diff
@@ -664,6 +677,10 @@ export function route(urlState) {
       state.get('nodeDetails'),
       dispatch
     );
+
+    if (urlState.contrastMode) {
+      dispatch(toggleContrastMode(true));
```

Comment:
I think `setContrastMode` might actually be a better name for this action (as I associate toggling with switching to the opposite state that wouldn't need any arguments).